src/shared/ptyfwd.h

```diff
@@ -12,7 +12,7 @@ typedef enum PTYForwardFlags {
         /* Continue reading after hangup? */
         PTY_FORWARD_IGNORE_VHANGUP         = 1 << 1,
 
-        /* Continue reading after hangup but only if we never read anything else? */
+        /* Continue reading after the initial hangup but only if we never read anything else? */
```

Comment:
Why does this matter? Can you elaborate? Ideally we'd make hangups idempotent if we haven't read anything anyways, and this changes that without sufficient reasoning?